### ISSUE TITLE
Dropdown for selecting assignee in merge request is always empty

### ISSUE THREAD

**Issue by michalhudecek** [PavlikPolivka/GitLabProjects#66]
![image](https://user-images.githubusercontent.com/1215304/28217150-66be9c92-68b4-11e7-87fc-bdecd87e0ae7.png)

Is there at least some workaround?

**Comment by PavlikPolivka** [comment]
Did you try to start typing the name? It's empty from until you start typing, there could be a lot of users so it's implemented this way.

**Comment by michalhudecek** [comment]
I see. When I start typing it freezes for a few seconds. Then it starts showing some names but they have nothing to do with the repo. Why is it not suggesting only people from the repo?

**Comment by PavlikPolivka** [comment]
It's using gitlab for searching users. As far as I know this was the only api option at the time of implementation. 

Now you can list users associated with project, it will list all the users without a search option, can be problematic for huge projects. I will try it tomorrow, it would definitely be more useful if the user list be more focused.

**Comment by michalhudecek** [comment]
That would be great!

**Comment by PavlikPolivka** [comment]
So I did two changes.
Now when the dialog window is initialized it will do a search for projects users with empty string query. It will grab some users and those will be used as initial model of the combo box. 
Not sure if it will grab all the project users as some projects will have a lot of users and gitlab api will limit the result set on search api call (not very well documented).
Second change is that when something is typed into the combo box it will search in projects users only. 
The slowdown is still there sometimes :( but due to the initial model most users will not hit it now.

I released this change now.
